src/components/photoGallery/PhotoGallery.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect } from "react";
+import "./PhotoGallery.scss"
+
+export const PhotoGallery: React.FunctionComponent = () => {
+
+  const [imageUrls, setImageUrls] = useState<string[]>();
+  const img_urls: string[] = [];
+
+        useEffect(() => {
+          const url = `https://api.nasa.gov/planetary/apod?count=50&api_key=DEMO_KEY`;
+          
+          const fetchData = async () => {
+            try {
+              const response = await fetch(url);
+              const json = await response.json();
+              json.forEach((item: {url: string}) => img_urls.push(item.url));  
+              setImageUrls(img_urls);
```

Comment:
Ah, I see why you were using `img_urls` above!
This should not be defined up there, and should be defined down here instead, but there's a better way using `map()`:
```suggestion
              setImageUrls(json.map(item => item.url));
```